### PR TITLE
Propose SampleMetadata and Dataset classes

### PR DESCRIPTION
**Pull request by jakebeal**
Address 183 by proposing SampleMetadata and Dataset classes, plus relabeling SampleArray.contents to SampleArray.initial_contents

### CODE REVIEW

File: labop/labop.ttl [hunk 1]
```diff
@@ -395,6 +451,32 @@ labop:fromSamples a owl:ObjectProperty ;
     rdfs:domain labop:SampleData ;
     rdfs:range labop:SampleCollection .
 
+labop:sampleDescriptions a owl:DatatypeProperty ;
```
Comment (bbartley) [inline labop/labop.ttl:454]: Should this be `owl:ObjectProperty` with `rdfs:range sbol:Implementation`?

Comment (jakebeal) [reply labop/labop.ttl:454]: I believe not, because it will be an xarray of Implementation URIs.

**Review by danbryce — CHANGES_REQUESTED** [review]
See my comments about naming Dataset attributes.

File: labop/labop.ttl [hunk 2]
```diff
@@ -395,6 +451,32 @@ labop:fromSamples a owl:ObjectProperty ;
...
+
+labop:data a owl:ObjectProperty ;
+    rdfs:label "data" ;
+    rdfs:comment """The data property indicates the SampleData used in a Dataset.""" ;
+    rdfs:domain labop:Dataset  ;
+    rdfs:range labop:SampleData  .
+
+
+labop:labels a owl:ObjectProperty ;
+    rdfs:label "labels" ;
```
[the remark this answers is not in the document]
Comment (jakebeal) [reply labop/labop.ttl]: Good point. I've changed it.

File: labop/labop.ttl [hunk 3]
```diff
@@ -235,6 +235,62 @@ labop:SampleData a owl:Class ;
...
+        """ ;
+    rdfs:subClassOf [ a owl:Restriction ;
+            owl:maxCardinality "1"^^xsd:nonNegativeInteger ;
+            owl:onProperty labop:data ],
+        [ a owl:Restriction ;
+            owl:maxCardinality "1"^^xsd:nonNegativeInteger ;
+            owl:onProperty labop:labels ],
+        [ a owl:Restriction ;
+            owl:allValuesFrom labop:SampleMetadata  ;
+            owl:onProperty labop:labels ],
```
[the remark this answers is not in the document]
Comment (jakebeal) [reply labop/labop.ttl]: As above, changed.

**Review by danbryce — APPROVED** [review]

**Pull request merged**